site/blocks/front-matter.scss

```diff
@@ -34,11 +34,11 @@
     cursor: pointer;
 
     &:hover {
-        border-color: darken($accent-vermillion, 10);
```

Comment:
The use of `darken()` seems to be [discouraged now](https://sass-lang.com/documentation/modules/color#darken):

> Because darken() is usually not the best way to make a color darker, it’s not included directly in the new module system. However, if you have to preserve the existing behavior, darken($color, $amount) can be written [color.adjust($color, $lightness: -$amount)](https://sass-lang.com/documentation/modules/color#adjust).

Reply:
Yup, exactly. That's why I removed it 🙂

Reply:
oh... "red" is "delete"? 🤦‍♂️😅